runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1232,32 +1232,38 @@ def DeleteCommands()
   delcommand Var
   delcommand Winbar
 
-  if exists('k_map_saved')
-    if !empty(k_map_saved) && !k_map_saved.buffer
+  if k_map_saved isnot null_dict
+    if !empty(k_map_saved) && k_map_saved.buffer
+      # pass
+    elseif !empty(k_map_saved) && !k_map_saved.buffer
       nunmap K
       mapset(k_map_saved)
     elseif empty(k_map_saved)
-      nunmap K
+      silent! nunmap K
     endif
-    k_map_saved = {}
+    k_map_saved = null_dict
   endif
-  if exists('plus_map_saved')
-    if !empty(plus_map_saved) && !plus_map_saved.buffer
+  if plus_map_saved isnot null_dict
```

Comment:
```suggestion
  if plus_map_saved != null
```
For reference
```
vim9script
var d1: dict<any> = null_dict
var d2: dict<any> = {}
echo 'is d1 null?' d1 == null
echo 'is d2 null?' d2 == null
```
outputs
```
is d1 null? true
is d2 null? false
```

Reply:
if you recalled some months ago what we argued in such null/null_obj topic which chris applied your PR,
i am confused now truly null or null_obj which was your taste? :sweat_smile:
anyway, `{} isnot null_dict` also is true, specially it is just keeping it same operation as `saved_mousemodel` did,
so i think ok.

Reply:
strings are the weirdest of them all when it comes to null handling. With strings you must use `isnot null_string`, as it says at  `:help null-anomalies`
```
Unlike the other containers, an uninitialized string is equal to null.
```

I left out some important cases from the example.
```vim
echo 'is d1 null_dict?' d1 == null_dict
echo 'is d2 nul_dictl?' d2 == null_dict
```
Both return `true`.

`!= null` is easier to understand than `isnot null_dict` (at least for me). It also looks like what you'd expect to see in most any other language; to me, that makes it easier to maintain.

As you say, what's currently there works; just not as maintainable (but that's only my opinion)